font-types/src/name_id.rs

```diff
@@ -0,0 +1,184 @@
+    /// with a space between “Version” and the number).
+    pub const VERSION_STRING: Self = Self(5);
+
+    /// PostScript name for the font; Name ID 6 specifies a string which is used to invoke a PostScript language font
+    /// that corresponds to this OpenType font. When translated to ASCII, the name string must be no longer than 63
+    /// characters and restricted to the printable ASCII subset, codes 33 to 126, except for the 10 characters '[', ']',
+    /// '(', ')', '{', '}', '<', '>', '/', '%'.
+    pub const POSTSCRIPT_NAME: Self = Self(6);
+
+    /// Trademark; this is used to save any trademark notice/information for this font. Such information should be based
+    /// on legal advice. This is distinctly separate from the copyright.
+    pub const TRADEMARK: Self = Self(7);
+
+    /// Manufacturer name.
+    pub const MANUFACTURER: Self = Self(8);
+
+    /// Designer; name of the designer of the typeface.
+    pub const DESIGNER: Self = Self(9);
+
+    /// Description; description of the typeface. Can contain revision information, usage recommendations, history, features, etc.
+    pub const DESCRIPTION: Self = Self(10);
+
+    /// URL Vendor; URL of font vendor (with protocol, e.g., http://, ftp://). If a unique serial number is embedded in the URL,
+    /// it can be used to register the font.
+    pub const VENDOR_URL: Self = Self(11);
+
+    /// URL Designer; URL of typeface designer (with protocol, e.g., http://, ftp://).
+    pub const DESIGNER_URL: Self = Self(12);
+
+    /// License Description; description of how the font may be legally used, or different example scenarios for licensed use.
+    /// This field should be written in plain language, not legalese.
+    pub const LICENSE_DESCRIPTION: Self = Self(13);
+
+    /// URL where additional licensing information can be found.
+    pub const LICENSE_URL: Self = Self(14);
+
+    /// Typographic Family name: The typographic family grouping doesn’t impose any constraints on the number of faces within it,
+    /// in contrast with the 4-style family grouping (ID 1), which is present both for historical reasons and to express style
+    /// linking groups.
+    pub const TYPOGRAPHIC_FAMILY_NAME: Self = Self(16);
+
+    /// Typographic Subfamily name: This allows font designers to specify a subfamily name within the typographic family grouping.
+    /// This string must be unique within a particular typographic family.
+    pub const TYPOGRAPHIC_SUBFAMILY_NAME: Self = Self(17);
+
+    /// Compatible Full (Macintosh only); On the Macintosh, the menu name is constructed using the FOND resource. This usually matches
+    /// the Full Name. If you want the name of the font to appear differently than the Full Name, you can insert the Compatible Full
+    /// Name in ID 18.
+    pub const COMPATIBLE_FULL_NAME: Self = Self(18);
+
+    /// Sample text; This can be the font name, or any other text that the designer thinks is the best sample to display the font in.
```

Comment:
```suggestion
    /// Sample text.
    ///
    /// This can be the font name, or any other text that the designer thinks is the best sample to display the font in.
```